pkg/csi/recover/recover.go

```diff
@@ -213,5 +217,15 @@ func (r *FuseRecover) eventRecord(point mountinfo.MountPoint, eventType, eventRe
 		return
 	}
 	glog.V(4).Infof("record to dataset: %s, namespace: %s", dataset.Name, dataset.Namespace)
-	r.Recorder.Eventf(dataset, eventType, eventReason, "Fuse recover %s succeed", point.MountPath)
+	switch eventReason {
+	case common.FuseRecoverSucceed:
+		r.Recorder.Eventf(dataset, eventType, eventReason, "Fuse recover %s succeed", point.MountPath)
+	case common.FuseRecoverFailed:
+		r.Recorder.Eventf(dataset, eventType, eventReason, "Fuse recover %s failed", point.MountPath)
+	}
+	// add warning event if point.count is larger than the threshold
+	if point.Count > r.recoverWarningThreshold {
+		r.Recorder.Eventf(dataset, corev1.EventTypeWarning, "TooManyRecovery", "Mountpoint %s has been mounted %v times", point.MountPath, point.Count)
+		glog.Warningf("TooManyRecovery: Mountpoint %s has been mounted %v times", point.MountPath, point.Count)
```

Comment:
I suggest changing `TooManyRecovery: Mountpoint %s has been mounted %v times`  to TooManyRecovery: Mountpoint %s has been recovered %v times,  please delete the pod accordingly`. Can we suggest which app pod should be deleted.

Reply:
There may be no information about app pod name and so on.

Reply:
> There may be no information about app pod name and so on.

I think pod uid is okay.